MEMBER_TRAVEL_FUND.MD

```diff
@@ -0,0 +1,41 @@
+## Purpose
+
+To establish and administer a fund for members of the Node.js
+Foundation Community Committee(CommComm) to travel and spread knowledge about and support the foundation
+and the Node.js Foundation projects.
+
+## Restrictions
+
+* Candidates must be CommComm Members of the Node.js project.
```

Comment:
I _think_ this should be rephrased to "Candidates must be a CommComm member, or a member of a team or Working Group overseen by CommComm", right?